src/main/java/hudson/plugins/jira/JiraMailAddressResolver.java

```diff
@@ -20,8 +20,18 @@
 public class JiraMailAddressResolver extends MailAddressResolver {
     private static final Logger LOGGER = Logger.getLogger(JiraMailAddressResolver.class.getName());
 
+    /**
+     * Boolean to disable the Jira mail address resolver.
+     *
+     * To disable set the System property "-Dhudson.plugins.jira.JiraMailAddressResolver.DISABLE=true"
+     */
+    public static final boolean DISABLE = Boolean.getBoolean(JiraMailAddressResolver.class.getName() + ".DISABLE");
```

Comment:
Since we are setting this argument through command line it shouldn't be modifiable. If I remove the `final`, then I can use a groovy script to set the value, and makes the argument useless.